app/bot.go

```diff
@@ -198,6 +198,113 @@ func (a *App) disableUserBots(userId string) *model.AppError {
 	return nil
 }
 
+func (a *App) notifySysadminsBotOwnerDeactivated(userId string) *model.AppError {
+	perPage := 25
+	botOptions := &model.BotGetOptions{
+		OwnerId:        userId,
+		IncludeDeleted: false,
+		OnlyOrphaned:   false,
+		Page:           0,
+		PerPage:        perPage,
+	}
+	// get owner bots
+	var userBots []*model.Bot
+	for {
+		bots, err := a.GetBots(botOptions)
+		if err != nil {
+			return err
+		}
+
+		if len(bots) == 0 {
+			break
+		}
+		userBots = append(userBots, bots...)
+		botOptions.Page += 1
```

Comment:
You can save a call by checking if `len(bots) < perPage`.

Reply:
Good catch.  I've incorporated that idea in both `for` loops